memory/libhugetlbfs.py

```diff
@@ -47,6 +47,11 @@ def setUp(self):
         # Check for basic utilities
         smm = SoftwareManager()
         detected_distro = distro.detect()
+        
+        if detected_distro.name == 'rhel':
+            if (detected_distro.version == '9' and detected_distro.release == '1'):
```

Comment:
this test case not support RHEL9-- onwards even  may you can use `skip` global decorate

Reply:
These scenarios are cancelled in earlier test cases rather than Skipping, hence we can continue with canceling the tests